client/src/pages/CreateContest/CreateContest.tsx

```diff
@@ -3,11 +3,41 @@ import Box from '@material-ui/core/Box';
 import Typography from '@material-ui/core/Typography';
 import Grid from '@material-ui/core/Grid';
 import Paper from '@material-ui/core/Paper';
+import { Moment } from 'moment';
+import axios from 'axios';
 import useStyles from './useStyles';
 
 export default function CreateContest():JSX.Element {
   const classes = useStyles();
 
+  const handleSubmit = (
+    {
+      title,
+      description,
+      prizeAmount,
+      deadlineDate,
+      images
+    }: {
+      title: string;
+      description: string;
+      prizeAmount: number;
+      deadlineDate: Moment;
+      images: Array<string>;
+    }
+  ) => {
+    const contest = {
+      title: title,
+      description: description,
+      prizeAmount: prizeAmount,
+      deadlineDate: deadlineDate.toDate(),
+      images: images,
+    };
+
+    axios.post('/contest', contest)
```

Comment:
Can we please create a helper function here to stay consistent with the code base?